apps/test/unit/templates/InlineMarkdownTest.js

```diff
@@ -75,4 +75,20 @@ describe('InlineMarkdown', () => {
       'block html is ignored'
     ).to.equal(true);
   });
+
+  it('will strip trailing newlines', () => {
+    const basicWrapper = shallow(
+      <InlineMarkdown
+        markdown={'some markdown with an _accidental_ new line \n'}
+      />
+    );
+
+    expect(
+      basicWrapper.equals(
+        <span>
+          some markdown with an <em>accidental</em> new line
+        </span>
+      )
+    ).to.equal(true);
```

Comment:
This test is also confusing since... that result is correct even if it didn't strip the newlines. newlines are allowed in markdown and should be ignored in this case in the first place. a comment is absolutely nice, here, to clarify the intent of the test and what failure (error in this case) is being considered.